### ISSUE TITLE
Archive issue discovered during thumbnail generation -- cannot find an image of less than 1MB in the first 20 images

### ISSUE THREAD

**Issue by TBRC-Travis** [buda-base/archive-ops#75]
- W8LS66751
- W17789
- W1KG12717
- W1FPL634
- W1FPL641
- W23746
- W1KG12716
- W1FPL647
- W00KG04002
- W2DB16431
- W00KG04068
- W1CZ2514
- W1KG12427
- W1FPL639
- W1KG5109
- W1FPL643
- W1AC16
- W00KG03991
- W5JW13
- W1KG18416
- W20817
- W20817
- W00KG03995
- W1KG16727
- W1KG3960
- W29621
- W1KG13825
- W1FPL633
- W1KG5724
- W1KG9250
- W23910
- W1FPL1738
- W1FPL642
- W27558
- W00KG04000
- W10919
- W22161
- W15575
- W1KG25128
- W5JW12
- W1CZ2278
- W2DB4589
- W1JM1
- W1KG14350
- W4CZ20865
- W5JW15
- W22393
- W1KG8321
- W3CN4061
- W00EGS1018179
- W18640
- W1PD45094
- W1KG12428
- W29619
- W00KG03999
- W29623
- W00KG04003
- W1KG12430
- W1KG12433
- W22314
- W1KG18421
- W3CN21845
- W1KG5950
- W1CZ2519
- W1KG12431
- W00KG03593
- W24054
- W1KG16154
- W00KG03996
- W1KG25001
- W00KG03994
- W1FPL644
- W1CZ2543
- W1FPL638
- W1KG13824
- W1KG12429
- W1KG5735
- W1KG6202
- W8LS32733
- W29620
- W1FPL637
- W00KG03846
- W1KG4592
- W00KG010266
- W1KG12709
- W1KG5966
- W20396
- W5JW16
- W5JW14
- W1FPL640
- W1KG18423
- W1KG6288
- W23690
- W2PD16797
- W1KG16728
- W00KG010090
- W1PD177857

**Comment by TBRC-Travis** [comment]
@eroux there are quite a few works you reported that don't have any images larger than our 400k size restriction:

W1AC16
W1FPL1738
W1KG12427
W1KG14350
W1KG16727
W1KG16728
W1KG18416
W1KG18421
W1KG18423
W1KG25001
W1KG25128
W1KG5966
W1KG9250
W1PD177857
W20817
W2DB4589
W2DB16431
W3CN21845
W3CN4061
W8LS32733
W8LS66751

Not sure why these were flagged during your thumbnail generation process.  Can you double check these in your logs?

**Comment by eroux** [comment]
Right, thanks for noticing! I indeed mixed two cases, one that you solved, and the one you spotted is a case where we have a released image group with a directory in our archive (with a `dimensions.json` file), but with no images or only the scan request images. The semantics of that case are quite unclear to me... does it mean the images are not synced yet? @karmagongde can you tell me more about when such a thing happens?

**Comment by karmagongde** [comment]
@eroux The folder with only the scan request images should not be moved for the processing. If these cases are mostly happened by the librarians. Any missing volume has remarked a note in the catalog records for the uses to know.
If it is possible to check the Archive and get a list of the folder with only the scan request image then we should remove them @TBRC-Travis . I will add a note in catalog records If the note is not added.

**Comment by TBRC-Travis** [comment]
Correct @karmagongde.  When I sync new works these "empty" image group folders with only scan request files are being deleted.  The list documented here is older works in the archive.  I will cleanup the directories in the above list to make sure these folders are deleted from the web images folder.

**Comment by eroux** [comment]
ah excellent, thanks! I think you can spot them easily by looking at folders with less than 3 images

**Comment by TBRC-Travis** [comment]
Precisely.  :)

**Comment by karmagongde** [comment]
Thank you Travis for taking care of these empty folders. Please let me know, any notes need to be added in the catalog record.

**Comment by TBRC-Travis** [comment]
@karmagongde will do!

**Comment by TBRC-Travis** [comment]
@karmagongde Here is the list of "empty" image group directories that only contain scan request files with no real images.  Please make sure the biblio note is updated for these works to reflect the missing volumes.

[empty-ig_dirs.txt](https://github.com/buda-base/archive-ops/files/4683868/empty-ig_dirs.txt)

**Comment by TBRC-Travis** [comment]
@eroux @xristy I've removed the image group dirs found in "empty-ig_dirs.txt" (see above comment).  If someone tries to access these on the website they will get an error now.  We'll need to somehow update exist db to remove these entries since there are still "ghost" View in Reader links displayed on the website.  I don't think I can update the entries using exist/rest/db/modules/admin/imagegroups-updater.xql so I think Chris will probably have to do it manually.

**Comment by karmagongde** [comment]
@TBRC-Travis Hi Travis,
I added the missing volume's note on catalog records. Many of them are Journal. 
Could you please keep W8LS32733-I8LS33439. It is a part of USAID images and looks like, images folder mistakenly moved to another folder. The folder name could be "ADA0015".

**Comment by TBRC-Travis** [comment]
@karmagongde I've restored W8LS32733-I8LS33439 as you've requested, though I don't see the images.  Please keep me updated if you find them.

**Comment by karmagongde** [comment]
@TBRC-Travis Hi Travis,
I got the images of W8LS32733-I8LS33439. My "A Better Finder Rename 9" is old and not working. Please rename the images to I8LS33439. I uploaded the folder on ReadyToProcess/USAID-2019. Thank you.

**Comment by TBRC-Travis** [comment]
@karmagongde Thanks Karma.  I see them now.  I'll take care of renaming them.
